<?xml version="1.0" encoding="UTF-8"?>
<<<<<<< HEAD
<testsuite hostname="Jayant" name="module9TestNG.RegistrationTest" tests="1" failures="1" timestamp="29 Mar 2018 19:08:53 GMT" time="0.002" errors="0">
  <testcase name="doREgister" time="0.002" classname="module9TestNG.RegistrationTest">
    <failure type="java.lang.AssertionError" message="Error not match expected:&amp;lt;xxxx&amp;gt; but was:&amp;lt;xxxx1&amp;gt;">
      <![CDATA[java.lang.AssertionError: Error not match expected:<xxxx> but was:<xxxx1>
	at module9TestNG.RegistrationTest.doREgister(RegistrationTest.java:18)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 14 stack frames]]>
    </failure>
  </testcase>
</testsuite>
=======
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-1ESRFEC" ignored="0" name="RegistrationTest" tests="1" failures="1" timestamp="14 Jan 2018 22:37:02 GMT" time="0.012" errors="0">
  <testcase name="doREgister" time="0.008" classname="module9TestNG.RegistrationTest">
    <failure type="java.lang.AssertionError" message="Error not match expected [xxxx] but found [xxxx1]">
      <![CDATA[java.lang.AssertionError: Error not match expected [xxxx] but found [xxxx1]at module9TestNG.RegistrationTest.doREgister(RegistrationTest.java:18)... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- doREgister -->
</testsuite> <!-- RegistrationTest -->
>>>>>>> b95f80cf2e33dc63fda04097a1702ad30d7f3615
